packages/expo-cli/package.json

```diff
@@ -116,7 +116,7 @@
     "slash": "1.0.0",
     "strip-ansi": "^6.0.0",
     "tar": "^6.0.5",
-    "tempy": "^0.3.0",
+    "tempy": "^0.7.1",
```

Comment:
Hmmm, why do we need `tempy` at all?
There's a built-in nodejs function that seems to return similar (if not the same) thing: https://nodejs.org/api/os.html#os_os_tmpdir

Reply:
@dsokal for me, it's just the nicer API and auto creation of folder and cleanup etc. few lines, but still